### ISSUE TITLE
Feedback on miniex 8 by Winnie Soon

### ISSUE THREAD

**Issue by siusoon** [a9neh/Anines-mini_ex#13]
No worry it is not a must if your program without an end. Usually a program without an end will indicate by a loop where an end process points back to an earlier stage. You can see a lot of our classmates produce flowcharts without an absolute end. Few comments on yours:

1/ I think the preload and display image can be combined into one. It makes only really minor differences for the audiences.
2/ What do you mean by draw background? From a user perspective, I just see an image display on a background. Then ellipses start to appear. (are you referring to the background 255? if yes I think this is something you can skip)
3/ A diamond uses to communicate a decision with yes/no answer (or true/false). 
4/ For the bottom part of your flow chart, no one really know the four cases. When I read your code, the four cases are more for the random movement of your ellipses right? 

I have made my own flowchart based on your code and visual outcome, just for your reference (not a model answer). If you want to put more details on your flow chart with the diamond decision box,  you may also consider to write 'if random set 1 -> return yes//no; if random set 2 -> return yes/no" etc. The four diamonds can be displayed horizontally under randomise ellipse movement. 

![dsc_0072](https://cloud.githubusercontent.com/assets/1446073/24609832/8b6130d8-187c-11e7-8fa7-8046a05ab308.jpeg)

**Comment by a9neh** [comment]
Yes I do see that the simple flowchart you made covers about everything, I just wanted to make a more detailed one to feel less lazy haha. But you're right, the preload and background part might not be that crucial and can easily be removed. I will remember this for the next time I make a flowchart, thank you for the feedback.
